public/data/FidalMathew.json

```diff
@@ -0,0 +1,18 @@
+{
+    "name": "Fidal Mathew",
+    "bio": "Frontend Developer | I love exploring and learning new technologies. ",
```

Comment:
```suggestion
    "bio": "Frontend Developer | I love exploring and learning new technologies.",
```

Reply:
**Please remove that extra space, otherwise LGTM.**

Thanks for the contribution :)
Happy Halloween🎃